AssortedCallouts/AssortedCallouts.csproj

```diff
@@ -1,169 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I don't think you meant to delete the entire CSPROJ file? This breaks the project, please reverse the commit

Reply:
I reversed the commit and added the csproj file without dependency paths.